pages/about/index.vue

```diff
@@ -2,20 +2,54 @@
   <div class="about-page">
     <breadcrumb :breadcrumb="breadcrumb" :title="pageTitle" />
     <page-hero v-if="heroCopy">
-      <h1>{{ pageTitle }}</h1>
-      <p>
-        {{ heroCopy }}
-      </p>
+      <div class="row">
+        <div class="col">
+          <h1>{{ pageTitle }}</h1>
+          <p>
+            {{ heroCopy }}
+          </p>
+          <button class="about-page-button">
+            <a href="https://commonfund.nih.gov/sparc/" target="_blank">More Info NIH Website</a>
+          </button>
+        </div>
+        <div class="col">
+          <img class="page-hero-img" src="@/static/images/about-hero.png" />
+        </div>
+      </div>
     </page-hero>
     <div class="page-wrap container">
       <div class="subpage">
-        <el-row type="flex" justify="center">
-          <el-col :row="24">
-            <!-- eslint-disable vue/no-v-html -->
-            <!-- marked will sanitize the HTML injected -->
-            <div v-html="parseMarkdown(copy)" />
-          </el-col>
-        </el-row>
+        <div class="row">
```

Comment:
Nit: Why not use `el-row`?

Reply:
@cameronbaney I was using `el-row` and `el-col` but they were giving me trouble with spacing and alignment, especially when it came to mobile and tablet viewports. I just found it a lot easier to use the traditional css flex attributes.

Reply:
Sounds good, thanks for the info.